sql/testdata/snapshot

```diff
@@ -0,0 +1,58 @@
+# Run a simple shell session in which a reader conflicts with an open txn in
+# SNAPSHOT isolation by means of a read, which is resolved by a Push. Prevents
+# regression of #2861.
```

Comment:
not put a context. The reason I did this is because I doubt I'll ever care
about this being related to that specific issue number, but I'll wind up
looking for tests that have to do with snapshot isolation.

On Mon, Nov 9, 2015 at 10:30 AM, Peter Mattis <notifications@github.com>
wrote:

> In sql/testdata/snapshot
> <https://github.com/cockroachdb/cockroach/pull/3052#discussion_r44287372>:
>
> > @@ -0,0 +1,58 @@
> > +# Run a simple shell session in which a reader conflicts with an open txn in
> > +# SNAPSHOT isolation by means of a read, which is resolved by a Push. Prevents
> > +# regression of #2861.
>
> What will we do with issues that have no simple context?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3052/files#r44287372>.
>